library/src/main/java/fr/tvbarthel/intentshare/IntentShare.java

```diff
@@ -94,6 +99,7 @@ private IntentShare(Context context) {
         packageWithExtraProvider = new ArrayList<>();
         this.listener = null;
         this.iconLoader = new AsyncIconLoader();
+        this.chooserTitle = context.getString(R.string.default_sharing_label);
```

Comment:
Add a custom prefix to all the resources of this library to avoid conflict in client project.

Proposition  "icl_default_sharing_label" for intent share library.